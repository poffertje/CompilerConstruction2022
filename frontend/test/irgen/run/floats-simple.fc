extern int printf(char[] format, ...);

int main() {
    float f = 4.0;
    printf("%.3f\n", f);                    // 4.000
    printf("%.3f\n", f / 1.3);              // 3.077
    return 0;
}
