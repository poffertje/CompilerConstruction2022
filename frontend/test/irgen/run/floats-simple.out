4.000
3.077
